spec/workers/fetch_graph_worker_spec.rb

```diff
@@ -5,19 +5,32 @@
   let(:worker) { described_class.new }
   let(:uri) { 'http://my.queryuri.com' }
   let(:user) { build(:user) }
+  let(:model) { create(:generic, title: ['foo'], creator: [controlled_val], depositor: user.email, id: 123) }
+  let(:controlled_val) { OregonDigital::ControlledVocabularies::Creator.new('http://opaquenamespace.org/ns/creator/ChabreWayne') }
 
+  # NOTE: THESE ARE XIT'd out until we know what we want to do with emailing.
   describe '#triplestore' do
-    it { expect(worker.triplestore).to be_a(TriplestoreAdapter::Triplestore) }
+    xit { expect(worker.triplestore).to be_a(TriplestoreAdapter::Triplestore) }
```

Comment:
Shouldn't need to be xit'ed

Reply:
The FetchGraphWorker doesnt actually use a triplestore anymore. So I'll just remove the test.